sphinx/source/demo.rst

```diff
@@ -56,5 +56,5 @@ The ``e2e`` test script takes several additional parameters, documented by passi
 
 .. rubric:: Footnotes
 
-.. [#log_location] The log location should be visible in the Python output. By default, node ``N`` will log to files ``out`` and ``err`` in ``/tmp/{USER}_{N}``
+.. [#log_location] The log location should be visible in the Python output. By default, node ``N`` will log to files ``out`` and ``err`` in ``CCF/build/ccf_workspace/end_to_end_scenario_{N}``
```

Comment:
s/ccf_workspace/ws/, we know it's CCF. Perhaps test_ws to make it clear it's tests.

Reply:
Oops. I renamed it in python but not here.